docs/modules/installation/pages/requirements.adoc

```diff
@@ -6,6 +6,10 @@
 {page-component-title} works with Grafana Server version *7.0* and up.
 It has been tested with Grafana versions up to 7.2 at the time of release.
 
+Note that the Grafana image renderer plugin depends on a project called https://github.com/puppeteer/puppeteer[Puppeteer] for an embedded headless Chrome/Chromium that has some unrequired dependencies.
```

Comment:
Are we saying that users should install Puppeter, or just the list of dependencies listed in the article on line 10?

Reply:
The Grafana renderer doesn't specifically require Puppeteer, but Puppeteer is a headless option (no GUI) that satisfies the requirements.  However, line 10 is referencing a dependency list of packages that Puppeteer requires but doesn't auto-install.  Users need to install those packages before installing Puppeteer.
This isn't immediately clear in the current wording here, and could be confusing to users.  (side note, I setup a docker instance with grafana and the renderer and that was really easy to deploy, but we can't assume all users are running within docker)

Reply:
Can we just give the same list of packages they mention without installing puppeteer?

Reply:
I take that back. According to the original Jira from last year, puppeteer was at some point a dependency of the grafana renderer.  If it isn't a dependency today, then maybe all of this is nonsense. :man_shrugging: